metricbeat/docs/modules/aerospike.asciidoc

```diff
@@ -41,8 +41,24 @@ metricbeat.modules:
   enabled: true
   period: 10s
   hosts: ["localhost:3000"]
+  # Aerospike Cluster Name
+  #cluster_name: myclustername
```

Comment:
Why cluster_name is now required? Also, irrespective of the return value of  `config.TLS.IsEnabled()`, the cluster name is set regardless. It was working previously without the cluster_name right?

Reply:
Hi, yes it did work before. By further reading the documentation of the Aerospike the ClusterName setting is actually used for:

> Expected cluster name. If populated, the clusterName must match the cluster-name field in the service section in each server configuration. This ensures that the specified seed nodes belong to the expected cluster on startup. If not, the client will refuse to add the node to the client's view of the cluster.

So it is not necessarily related to TLS. Indeed, the example of the aerospike documentation with TLS and Java Client https://aerospike.com/docs/server/operations/configure/network/tls/mtls_java does not use the parameter.

So I think, it is okay if the setting is set regardless of TLS. I am open to remove it from the PR if we want to keep the scope of the PR tight.

Reply:
No, it is fine. Let's keep it.